.vscode/launch.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
@OlegNovosad could you add this file to .gitignore

Reply:
will add in next PR